datastream-server/src/main/java/com/linkedin/datastream/server/assignment/StickyPartitionAssignmentStrategy.java

```diff
@@ -187,20 +209,15 @@ public StickyPartitionAssignmentStrategy(Optional<Integer> maxTasks, Optional<In
     Validate.isTrue(currentAssignment.size() > 0,
         "Zero tasks assigned. Retry leader partition assignment.");
 
+    DatastreamGroup datastreamGroup = datastreamPartitions.getDatastreamGroup();
     String dgName = datastreamPartitions.getDatastreamGroup().getName();
```

Comment:
Since you are adding additional variable `datastreamGroup` use that to get `dgName`